cosa/cli-spec.md

```diff
@@ -0,0 +1,64 @@
+# CoreOS Assemlber Config Specification
+- Subject ID: COSA.20200804
+- Superceeds: None
+- Conflicts: None
+
+## Summary
+
+The RHCOS development pipelines use a "jobspec" to describe the pipeline runs.  This enhancement proposes a file-based interface based loosely on the RHCOS jobspec.
+
+## Goals
+
+1. Provide a file-based interface into COSA
+1. Prep for breaking up the Pipeline monoliths
+1. Provide a migration path for RHCOS pipelines
+1. Prepare for pipeline Nirvana (where *COS pipelines can live in peace and harmony)
+
+## Why?
+
+The innovation of the RHCOS JobSpec was it provided a clear seperation of configuration from the code; the logic was conflated. Further it the jobspec allowed the same logic to be run with different inputs.
```

Comment:
`Further it the jobspec`

Is there a better word or wording to use?

Reply:
Fixed up.